packages/core/src/configuration.ts

```diff
@@ -287,7 +301,8 @@ export const axis = {
 	ticks: {
 		number: 7,
 		rotateIfSmallerThan: 30
-	}
+	},
+	paddingRatio: 0.1
```

Comment:
What does this do now? Does it clash with `addSpaceOnEdges`?

Reply:
<img width="462" alt="Screenshot 2020-02-26 at 20 04 08" src="https://user-images.githubusercontent.com/5892387/75378028-3512f500-58d3-11ea-882d-70d0f1f0985d.png">

We just improved the old logic to add padding, where we use the aspect ratio to calculate the padding we should add to the domain, based on domain length.